fe/fe-core/src/main/java/org/apache/doris/statistics/StatisticsJobManager.java

```diff
@@ -80,16 +80,24 @@ public Map<Long, StatisticsJob> getIdToStatisticsJob() {
     }
 
     public void createStatisticsJob(AnalyzeStmt analyzeStmt) throws UserException {
-        // step1: init statistics job by analyzeStmt
-        StatisticsJob statisticsJob = StatisticsJob.fromAnalyzeStmt(analyzeStmt);
-        writeLock();
-        try {
-            // step2: check restrict
-            checkRestrict(analyzeStmt.getDbId(), statisticsJob.getTblIds());
-            // step3: create it
-            createStatisticsJob(statisticsJob);
-        } finally {
-            writeUnlock();
+        // The current statistics are only used for CBO test,
+        // and are not available to users. (work in progress)
+        // TODO(wzt): Further tests are needed
+        if (Config.enable_cbo_statistics) {
```

Comment:
Set it as a session variables rather than an options in configuration file

Reply:
sure